.github/ISSUE_TEMPLATE/1-bug.md

```diff
@@ -48,3 +48,11 @@ confidential information out.
 
 [1]: https://gist.github.com
 -->
+
+<!-- 
+NOTE: If your bug is meets one of the following criteria, and is not recieving proper attention, please notify the team directly by commenting @TEAM-MEMBER, or messaging @TEAM-MEMBER on our [Slack](https://cdk.dev/):
```

Comment:
I'm not super comfortable with the phrasing here:

- `@TEAM-MEMBER` looks like a literal term to use... But folks would need to message *some* team member... Now... how do they identify team members?
  - Not too sure how to address this all the while not violating some policy
- `our [Slack]` seems to imply we own that slack, which is not true... 
  - Proposed a simple re-phrasing around this below

```suggestion
NOTE: If your bug is meets one of the following criteria, and is not recieving proper attention, please notify the team directly by commenting @TEAM-MEMBER, or messaging @TEAM-MEMBER on the [CDK.dev Slack](https://cdk.dev/):
```